tests/Doctrine/Tests/DBAL/Functional/BlobTest.php

```diff
@@ -124,4 +175,17 @@ private function assertBlobContains($text)
         self::assertInternalType('resource', $blobValue);
         self::assertEquals($text, stream_get_contents($blobValue));
     }
+
+    private function assertClobContains($text)
+    {
+        $rows = $this->_conn->fetchAll('SELECT * FROM blob_table');
+
+        self::assertCount(1, $rows);
+        $row = array_change_key_case($rows[0], CASE_LOWER);
```

Comment:
You can use `$conn->query()->fetchAll(FetchMode::COLUMN)` and avoid the case conversion (it's not part of the SUT).

Reply:
D'oh! I thought you confused `COLUMN` and `FETCH_ASSOC`, but now it makes sense.